pkg/services/libraryelements/database.go

```diff
@@ -433,21 +432,17 @@ func (l *LibraryElementService) getAllLibraryElements(c context.Context, signedI
 			builder.Write(" UNION ")
 		}
 		builder.Write(selectLibraryElementDTOWithMeta)
-		builder.Write(", dashboard.title as folder_name ")
-		builder.Write(", dashboard.uid as folder_uid ")
+		builder.Write(", le.folder_uid as folder_name ")
```

Comment:
This is different functionality

Reply:
This results in the folder displayed on the list of library elements to be the uid rather than the title:

<img width="189" alt="Screenshot 2025-01-17 at 3 26 44 PM" src="https://github.com/user-attachments/assets/c52862fa-685f-409c-b3e1-b6678b031c3f" />